### ISSUE TITLE
[Tests] Investigate how hyperparameters impact learning

### ISSUE THREAD

**Issue by matkrak** [matkrak/enlargeme#14]
Compare efficiency, speed, memory used for different conv_kernel sizes, different filters_no and learning rate.
